src/containers/reducers/book.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-return-assign */
```

Comment:
## Move the reducer directory
- [ ] The `reducer` directory needs to be moved to the `src` directory. You only need to have the `bookForm.js` and the `bookList.js` components in this directory.